src/mesh/smoothing.rs

```diff
@@ -74,12 +74,13 @@ pub fn laplacian_smoothing(
         }
     }
 
-    // FIXME: Calculate smooth normals for the result once we support them
```

Comment:
Isn't the correct thing to do to tell the function what kind of normals we want it to compute? Or are smooth normals always desired and sharp normals never?

We don't need to expose this choice to the UI, but having it would also allow you to not modify existing tests and snapshots, but only add a couple of ones for new behavior.

Reply:
I need to have a better look how deeply integrated is the normal calculation into the functions. If it is easy, I'm in for giving the choice. We can also expose it in the UI (of we already support boolean inputs).